index.bs

```diff
@@ -1153,7 +1154,9 @@ The dictionary SHALL have the following attributes:
 :: The number of bytes sent on the QUIC connection, including retransmissions.
    Does not include UDP or any other outer framing.
 : <dfn for="WebTransportStats" dict-member>packetsSent</dfn>
-:: The number of packets sent on the QUIC connection, including retransmissions.
+:: The number of packets sent on the QUIC connection, including those that are determined to have been lost.
+: <dfn for="WebTransportStats" dict-member>packetsLost</dfn>
+:: The number of packets lost on the QUIC connection (does not monotonically increase).
```

Comment:
```suggestion
:: The number of packets lost on the QUIC connection (does not monotonically increase, because packets that are declared lost can subsequently be found).
```